frontend/test/__support__/e2e/helpers/e2e-misc-helpers.js

```diff
@@ -210,3 +210,13 @@ function dashboardHasQuestions(cards) {
     return false;
   }
 }
+
+export function interceptIfNotPreviouslyDefined({ method, url, alias } = {}) {
+  const aliases = Object.keys(cy.state("aliases"));
+
+  const isAlreadyDefined = aliases.find(a => a === alias);
+
+  if (!isAlreadyDefined) {
+    cy.intercept(method, url).as(alias);
+  }
+}
```

Comment:
I wasn't sure about the naming here, but this (internal) helper can be quite useful in the future and should remove duplicate intercepts.